debian/copyright

```diff
@@ -1,30 +1,36 @@
-This is Grml's GnuPG keyrings of archive keys.
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 
-This package was originally put together by Michael Vogt
-<mvo@debian.org> for Debian and has been adapted to grml.org by
-Alexander Wirt <formorer@grml.org> in 2007. Minor updates were done by
-Michael Prokop <mika@grml.org> and Chris Hofstaedtler <ch@grml.org>
-in the years 2007 to 2025.
+Files:
+ *
+Copyright:
+ 1998-2003 James Troup <james@nocrew.org>
+ 2006 Michael Vogt <mvo@debian.org>
+ 2007 Alexander Wirt <formorer@grml.org>
+ 2007-2025 Michael Prokop <mika@grml.org>
+ 2011-2025 Chris Hofstaedtler <ch@grml.org>
+License: GPL-2+
 
-The keys in the keyrings don't fall under any copyright.  Everything else in
-the package is covered by the GNU GPL.
+Files:
+ keyrings/*
+Copyright:
+ none
```

Comment:
hm this triggers `missing-field-in-dep5-copyright` in lintian AFAICS, should we override this in lintian?

Reply:
Ah, or just add a License: GPL-2+? (I've done that for now, but I'm happy to change it to something else, add overrides or whatever)

Reply:
ACK, good idea!